api/_mixins/LinkStyle__SVGStyleElement.json

```diff
@@ -0,0 +1,55 @@
+{
+  "api": {
+    "SVGStyleElement": {
+      "sheet": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGStyleElement/sheet",
+          "spec_url": "https://drafts.csswg.org/cssom/#dom-linkstyle-sheet",
+          "support": {
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": "18"
+            },
+            "edge": {
+              "version_added": "12"
+            },
+            "firefox": {
+              "version_added": "1.5"
+            },
+            "firefox_android": {
+              "version_added": "4"
+            },
+            "ie": {
+              "version_added": "9"
+            },
+            "opera": {
+              "version_added": "≤12.1"
+            },
+            "opera_android": {
+              "version_added": "≤12.1"
+            },
+            "safari": {
+              "version_added": "3"
+            },
+            "safari_ios": {
+              "version_added": "1"
+            },
+            "samsunginternet_android": {
+              "version_added": "1.0"
+            },
+            "webview_android": {
+              "version_added": "1"
```

Comment:
Would this pass the lint if it was in SVGStyleElement.json? You changed lots of data to <=37 so I wonder if this can really be right?

Reply:
Oops, was on autopilot for this one -- fixed!

Reply:
Thanks!